src/LightInject/LightInject/LightInject.cs

```diff
@@ -4146,16 +4162,12 @@ private Action<IEmitter> CreateEmitMethodBasedOnClosedGenericServiceRequest(Type
             }
 
             var serviceRegistration = new ServiceRegistration
-                                                          {
-                                                              ServiceType = closedGenericServiceType,
-                                                              ImplementingType =
-                                                                  closedGenericImplementingType,
-                                                              ServiceName = serviceName,
-                                                              Lifetime =
-                                                                  CloneLifeTime(
-                                                                      openGenericServiceRegistration
-                                                                  .Lifetime)
-                                                          };
+            {
+                ServiceType = closedGenericServiceType,
+                ImplementingType = closedGenericImplementingType,
+                ServiceName = serviceName,
+                Lifetime = CloneLifeTime(openGenericServiceRegistration.Lifetime) ?? DefaultLifetime
```

Comment:
Same here, not sure if DefaultLifetime makes sense here, as I don't 100% understand what this method does.